x-pack/plugins/maps/public/actions/tooltip_actions.ts

```diff
@@ -56,9 +56,11 @@ export function closeOnHoverTooltip() {
 }
 
 export function openOnHoverTooltip(tooltipState: TooltipState) {
-  return {
-    type: SET_OPEN_TOOLTIPS,
-    openTooltips: [tooltipState],
+  return (dispatch: Dispatch) => {
```

Comment:
Can you revert to original form and return action instead of returning a thunk that is nothing more than a wrapper around the action. Thunk is not needed since there is only a single action.